tracer_options.go

```diff
@@ -73,14 +73,21 @@ func (tracerOptions) PoolSpans(poolSpans bool) TracerOption {
 	}
 }
 
-// HostIPv4 creates a TracerOption that identifies the current service/process.
-// If not set, the factory method will obtain the current IP address.
+// HostIPv4 is deprecated, please use HostIP instead.
 func (tracerOptions) HostIPv4(hostIPv4 uint32) TracerOption {
 	return func(tracer *tracer) {
 		tracer.hostIPv4 = hostIPv4
 	}
 }
 
+// HostIP creates a TracerOption that identifies the current service/process.
```

Comment:
Just thinking - what is the use case where someone **must** provide the IP explicitly instead of letting tracer get it from `net`? Maybe we should minimize the API surface, can always add it back if someone really needs it in the future.